goblet/resources/scheduler.py

```diff
@@ -26,7 +26,7 @@ class Scheduler(Handler):
     def register_job(self, name, func, kwargs):
         schedule = kwargs["schedule"]
         kwargs = kwargs.pop("kwargs")
-        timezone = kwargs.get("timezone", "UTC")
+        timezone = kwargs["timezone"]
```

Comment:
`timezone = kwargs["timezone"]` should be set after `schedule` and before we pop kwargs and reset it. Not the cleanest code I admit, so appreciate you digging through it. will think through better naming in the future i think instead of passing kwargs everywhere.

Reply:
ah thanks for catching that :)